File: app/src/main/java/tk/lorddarthart/rxnewstestapp/Item.java
package tk.lorddarthart.rxnewstestapp;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Item {
    @SerializedName("tittle")
    @Expose
    private String title;
    @SerializedName("desc")
    @Expose
    private String desc;
    @SerializedName("date")
    @Expose
    private String date;
    @SerializedName("pic")
    @Expose
    private String pic;

    String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    String getPic() {
        return pic;
    }

    public void setPic(String pic) {
        this.pic = pic;
    }
}

File: app/src/main/java/tk/lorddarthart/rxnewstestapp/application/model/Item.kt
package tk.lorddarthart.rxnewstestapp.application.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Item {
    @SerializedName("tittle")
    @Expose
    internal var title: String? = null
    @SerializedName("desc")
    @Expose
    internal var desc: String? = null
    @SerializedName("date")
    @Expose
    internal var date: String? = null
    @SerializedName("pic")
    @Expose
    internal var pic: String? = null
    @SerializedName("link")
    @Expose
    internal var link: String? = null
}
